app/controllers/api/functions.php

```diff
@@ -582,45 +579,18 @@
             'deploy' => ($deploy === 'true'),
         ]));
 
-        $usage
-            ->setParam('storage', $deployment->getAttribute('size', 0))
-        ;
-
-        // Send start build reqeust to executor using /v1/deployment
-        $function = $dbForProject->getDocument('functions', $functionId);
-
-        $ch = \curl_init();
-        \curl_setopt($ch, CURLOPT_URL, "http://appwrite-executor/v1/deployment");
-        \curl_setopt($ch, CURLOPT_POST, true);
-        \curl_setopt($ch, CURLOPT_POSTFIELDS, json_encode([
+        // Enqueue a message to start the build
+        Resque::enqueue('v1-builds', 'BuildsV1', [
```

Comment:
We have constants for the in the event class right?

Reply:
done 👍